### DISCUSSION TITLE
Specify Config file.

### DISCUSSION THREAD

**Discussion by XPAULUSX** [nuxt/nuxt#16554] [General]
Hi Guys,

In Nuxt 2 we used to be able to have multiple configuration files and switch to whichever using the following:

![image](https://user-images.githubusercontent.com/47896035/197541326-51931868-9e7d-4573-8368-ced5784de682.png)

Is the same functionality available within Nuxt 3?  I cant seem to get this working.

Thanks

Paul

**Comment by huang-julien** [comment]
I was looking for this too. it's currently not possible, feel free to open an issue for a feature request  🙂
